src/components/pricing/PricingBoxes.tsx

```diff
@@ -114,99 +116,74 @@ const plans: PricingBoxProps[] = [
+const PricingBoxes = ({ isRendered, changeIsRendered }: PricingBoxesProps) => {
     return <Styled>
         <section className="pricing">
             <div className="grey-container container--1">
                 <div className="row">
                     <h1 className="sub">Plans &amp; Pricing</h1>
-                    <div style={{
-                        display: 'flex',
-                        justifyContent: 'center'
-                    }}>
-                        <Tab style={{
-                            backgroundColor: selfHosted ? colors.offWhite2 : colors.white,
-                        }}
-                            onClick={() => setSelfHosted(false)}
-                        >Cloud</Tab>
-                        <Tab style={{
-                            backgroundColor: selfHosted ? colors.white : colors.offWhite2,
-                        }}
-                            onClick={() => setSelfHosted(true)}
-                        >Self-Hosted</Tab>
-                    </div>
+                    <Tabs
+                        isRendered={isRendered}
+                        changeIsRendered={changeIsRendered}
+                    />
                 </div>
             </div>
             <PricingContainer>
-                <div
-                    className={`pricing__boxes ${selfHosted ? 'hide' : 'show'}`}
-                >
-                    {plans.map(
-                        (plan, i) => <PricingBox
-                            key={i}
-                            {...plan}
-                        />
-                    )}
-                </div>
-                <div
-                    className={`pricing__boxes ${selfHosted ? 'show' : 'hide'}`}
-                >
-                    {selfHostedPlans.map(
-                        (plan, i) => <PricingBox
-                            key={i}
-                            {...plan}
-                        />
-                    )}
+                <div className="cards">
+                    <div
+                        className={`pricing__boxes ${isRendered ? 'hide' : 'show'}`}
+                    >
+                        {plans.map(
+                            (plan, i) => <PricingBox
+                                key={i}
+                                {...plan}
+                            />
+                        )}
+                    </div>
+                    <div
+                        className={`pricing__boxes ${isRendered ? 'show' : 'hide'}`}
+                    >
+                        {selfHostedPlans.map(
+                            (plan, i) => <PricingBox
+                                key={i}
+                                {...plan}
+                                isTitleOutside={true}
+                            />
+                        )}
+                    </div>
                 </div>
+                <p>Do you still have questions? Please, <Link to="/contact/">Get in Touch.</Link></p>
```

Comment:
```suggestion
                <p>Do you have any questions? Please, <Link to="/contact/">Get in Touch.</Link></p>
```

@ChristinFrohne Can we remove the  IMO it is not required here and does not make much sense. (Sorry If I misunderstood this I don't know English well).

Reply:
Sure that works too.